OpenSim/Simulation/Model/Marker.h

```diff
@@ -45,7 +45,13 @@ class OSIMSIMULATION_API Marker : public Station {
     OpenSim_DECLARE_CONCRETE_OBJECT(Marker, Station);
 
 class Body;
-
+public:
+    //==============================================================================
+    // PROPERTIES
+    //==============================================================================
+    OpenSim_DECLARE_PROPERTY(fixed, bool,
+        "Flag(true or false) specifying whether or not a marker "
+        "should be kept fixed in the marker placement step.  i.e. If false, the marker is allowed to move.");
```

Comment:
Thanks @aseth1 I'll reword per your suggestion but I'll try to avoid hard-coding class names (ScaleTool, MarkerPlacer) in case the functionality moves around or is replicated somewhere else. Current wording is from 3.3 verbatim.